data/data/bootstrap/systemd/units/systemd-journal-gatewayd.service.d/certs.conf

```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStart=
```

Comment:
Should this line be dropped?

Reply:
All systemd directives act as lists and each entry adds another item to the list. Simple services (like journal-gatewayd) can only have a single item for ExecStart, so we have to clear the list with an empty entry.